ecl/hql/hqlexpr.cpp

```diff
@@ -9500,7 +9500,11 @@ class VirtualSymbolReplacer : public QuickHqlTransformer
         switch (op)
         {
         case no_unboundselect:
-            throwUnexpected();
+            //If --fastsyntx is used it is possible for modules to no be fully bound.
```

Comment:
minor spelling: ..for modules to **no** be fully bound. should be for modules to **not** be fully bound.

Reply:
Also missing a in fastsyntax